docs/content/installation/other/helm.md

```diff
@@ -176,6 +176,33 @@ pgo client version 4.4.0-beta.1
 pgo-apiserver version 4.4.0-beta.1
 ```
 
+## Metrics Chart
+
+The PostgreSQL Operator metrics infrastructure can be installed using a separate
+Helm chart. This chart can install or uninstall metrics in a namespace where
+the operator is currently running. The metrics installer runs similarly to the
```

Comment:
> This chart can install or uninstall metrics in a namespace where
> the operator is currently running

Couldn't it be installed in a separate namespace as well, e.g. if `metrics_namespace` has a different namespace (or the Helm artifacts themselves if another namespace is specified when running `helm install metrics`.

Reply:
Yeah that is true. What about this?
> This chart can install or uninstall metrics in a provided namespace.

Reply:
I ended up going with something else entirely when i committed the change
>This chart can install or uninstall the Grafana and Prometheus deployments that are part of the metrics stack.